hash/bbc_flop_32016.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
 
-<!-- BBC Micro - 32016 Co-Processor Discs -->
+<!-- Acorn ABC - 32016 Co-Processor Discs -->
 
 <!-- Loading Instructions:
 
       Hold down the SHIFT key and press and release the BREAK key.
 -->
 
-<softwarelist name="bbc_flop_32016" description="32016 Co-Processor Discs">
+<softwarelist name="bbc_flop_32016" description="Acorn ABC 32016 Co-Processor discs">
```

Comment:
These are for any Acorn machine with a 32106 co-processor attached, not just the Acorn ABC range.